install/prereqs.rst

```diff
@@ -67,6 +67,14 @@ If you wish to follow the instructions for :doc:`lsstsw` (recommended for some d
 
    sudo apt-get update && sudo apt-get install git
 
+Tip: to display images in DS9, installing ``xpa-tools`` with the following statement
+might be necessary in some cases:
```

Comment:
I suspect it is necessary in all cases, no? xpa is required for sending images to ds9, and it's likely not installed on most setups (unless they already have been using ds9).

Reply:
I made it more specific that the cases being referred to are `xpa`-related errors displaying images in DS9, and that this specific install statement might be needed (i.e., it is not that `xpa-tools` might be needed, it will, but that they might need to be installed with this line). Is my understanding of the situation.